app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
Brackets are required if the string contains more than 79 characters and you want to format it according to the guidelines PEP8